larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -669,6 +670,7 @@ StatusCode MasterAlgorithm::Copy(const Pandora *const pPandora, const CaloHit *c
     // ATTN Parent of calo hit in worker is corresponding calo hit in master
     parameters.m_pParentAddress = static_cast<const void*>(pCaloHit);
     parameters.m_larTPCVolumeId = pLArCaloHit->GetLArTPCVolumeId();
+    parameters.m_daughterVolumeId = (m_larCaloHitVersion>1) ? pLArCaloHit->GetDaughterVolumeId() : 0;
```

Comment:
Spaces around `>`.

See later comment about default value for daughter volume id.